lib/Mandel/Collection.pm

```diff
@@ -33,7 +33,9 @@ sub create {
   my $cb = ref $_[-1] eq 'CODE' ? pop : undef;
   my $self = shift;
 
-  $self->_new_document(shift || undef, 0);
+  my $doc = $self->_new_document(shift || undef, 0);
+  $doc->_validate_fields;
```

Comment:
I suggest changing that method to `return $self`, so you can do:

```
return $self->_new_document(shift || undef, 0)->validate_fields;
```

Reply:
Also done